netdata-9999.ebuild

```diff
@@ -59,34 +56,59 @@ pkg_setup() {
 	enewuser ${PN} -1 -1 / ${PN}
 }
 
-src_unpack() {
-	if [[ ${PV} == "9999" ]] ; then
-		git-r3_src_unpack
-	fi
-	default
-}
-
 src_prepare() {
-	if [[ ${PV} == "9999" ]] ; then
-		eautoreconf
-	fi
 	default
+	[[ ${PV} == "9999" ]] && eautoreconf
 }
 
 src_configure() {
 	econf \
 		--localstatedir=/var \
+		--with-user=${NETDATA_USER} \
 		$(use_enable nfacct plugin-nfacct) \
-		$(use_with zlib) \
-		--with-user=${PN} \
-		|| die "configure failed"
+		$(use_with compression zlib)
 }
 
 src_install() {
 	default
 
-	fowners ${PN} /var/log/netdata
+	fowners ${NETDATA_USER} /var/log/netdata
+
+	chown -Rc ${NETDATA_USER} "${ED}"/usr/share/${PN} || die
```

Comment:
Hmm, that's not part of this patch. looks like PR has sucked in another commit. I'll sort that out!